app/controllers/admin/action_pages_controller.rb

```diff
@@ -28,7 +28,7 @@ def update_featured_pages
 
   def new
     @actionPage = ActionPage.new
-    @actionPage.partner = Partner.new
+    @actionPage.partners = Partner.all
```

Comment:
This should be fixed relatively soon -- new action pages are initialized with partnerships to everyone.